cluster/gce/gci/mounter/mounter

```diff
@@ -31,7 +31,7 @@ function gc {
     # Rkt pods end up creating new copies of mounts on the host. Hence it is ideal to clean them up right away.
     attempt=0
     until [ $attempt -ge 5 ]; do
-	${RKT_BINARY} gc --grace-period=0s &> /dev/null && break
```

Comment:
`1` might be of concern. Do we have any data to back up that concern?
I don't get `2`. Why is umount failing if other shared recursive mounts exist? Is the `umount` syscall failing or is it our own custom logic?